starter/app/build.gradle

```diff
@@ -48,6 +48,8 @@ dependencies {
 
     // Constraint Layout
     implementation "androidx.constraintlayout:constraintlayout:$constraintLayoutVersion"
+    implementation"androidx.constraintlayout:constraintlayout:2.1.4"
+    implementation"androidx.constraintlayout:constraintlayout-compose:1.0.1"
```

Comment:
please update `constraintLayoutVersion` to `2.1.4` . I don't think you need `constraintlayout-compose`